server/models/interfaces.ts

```diff
@@ -405,6 +409,11 @@ export interface GetSnapshot {
     successful: number;
     failed: number;
   };
+  restore_aliases?: boolean;
+  ignore_unavailable?: boolean;
+  rename_pattern?: string;
+  rename_replacement?: string;
+  partial?: boolean;
   metadata?: {
```

Comment:
Added these optional elements as they aren't always set/needed but for restoring.